README.md

```diff
@@ -9,6 +9,23 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
   git clone https://github.com/thewca/worldcubeassociation.org
   cd worldcubeassociation.org
   ```
+
+We can run the application in two ways:
+- [Run using Docker](#run-using-docker)
+- [Run directly with Ruby](#run-directly-with-ruby-lightweight-also-only-runs-the-rails-portions-of-the-site)
+
+### Run using Docker
+
+- Install [Docker](https://docs.docker.com/get-docker/) (remember to complete the [Linux post-install steps](https://docs.docker.com/engine/install/linux-postinstall/) if you're on Linux)
+- Install docker-compose. The best way to get an up-to-date version is to get it from their [releases page](https://github.com/docker/compose/releases)
+- Navigate into the repository's main directory (`worldcubeassociation.org`)
+- To start the server at `http://localhost:3000`, run `docker-compose up` and to bring it down, `docker-compose down` (or just press ctrl + c in the same terminal)
+- If you want to run the php part of the website as well, run `docker compose -f docker-compose.yml -f docker-compose.php.yml up`
+- To run tests, run `docker-compose exec wca_on_rails bash -c "RAILS_ENV=test bin/rake db:reset && RAILS_ENV=test bin/rake assets:precompile && bin/rspec"`
+- If you're using Visual Studio Code to develop, you can [attach it to the Docker container](https://code.visualstudio.com/docs/remote/containers) so that your extensions can take advantage of the Ruby environment and so the terminal runs from inside the container
+
+### Run directly with Ruby (lightweight, also only runs the Rails portions of the site)
```

Comment:
The "only Rails portion" part is obsolete now that we don't have PHP anymore. There is only Rails and running Rails runs the entire website.

Reply:
Oh yes, so is line 23 also not needed anymore? Looks like `docker-compose.php.yml` is still there.